### ISSUE TITLE
Set the configuration property typeKey to null value  in AerospikeDataProperties for the AerospikeTypeAliasAccessor bean

### ISSUE THREAD

**Issue by mrozk** [aerospike-community/spring-boot-data-aerospike#23]
Hi! For now the only way to setup AerospikeTypeAliasAccessor with typeKey=null is to redeclare bean. 
```
@Bean
    public AerospikeTypeAliasAccessor aerospikeTypeAliasAccessor() {
        //we do not want to save @class field with type of document into Aerospike
        return new AerospikeTypeAliasAccessor(null);
    }
```
We can't send typeKey=null  through configuration properties because the configuration spring.data.aerospike.typeKey=null is taking  all values as string. Let's discuss the most appropriate variant for setting null on this configuration, and I would contribute this. It can be placeholder @null for example.

**Comment by Aloren** [comment]
Setting it to `spring.data.aerospike.typeKey=~` in yaml also doesn't work?

**Comment by Aloren** [comment]
Okay, I've checked both `~` and `null` values -- both of them are parsed as an empty string. I've looked through the internet and this is what I found: https://github.com/spring-projects/spring-framework/issues/19986
I suppose, this behaviour won't be changed in the nearest future, so we definitely need to have a workaround for that.

I was thinking about some dirty hack like: 
`return new AerospikeTypeAliasAccessor(typeKey.equals("") ? null : typeKey);`. Need to discuss with the teammates.
